packages/eas-cli/src/credentials/ios/appstore/authenticate.ts

```diff
@@ -47,37 +57,40 @@ export type AuthCtx = {
   /**
    * Can be used to restore the Apple auth state via apple-utils.
    */
-  cookies?: Session.AuthState['cookies'];
+  authState?: Session.AuthState;
 };
 
+export function getRequestContext(ctx: AuthCtx): RequestContext {
+  assert(ctx.authState?.context, 'Apple request context must be defined');
+  return ctx.authState.context;
+}
+
 async function authenticateWithExperimentalAsync(options: Options = {}): Promise<AuthCtx> {
   const { appleId, appleIdPassword } = await requestAppleCredentialsAsync(options);
   log(`Authenticating to Apple Developer Portal...`); // use log instead of spinner in case we need to prompt user for 2fa
 
   try {
     // TODO: The password isn't required for apple-utils. Remove the local prompt when we remove traveling Fastlane.
-    const authContext = await Auth.loginAsync({
+    const authState = await Auth.loginAsync({
       username: appleId,
       password: appleIdPassword,
       cookies: options.cookies,
+      teamId: options.teamId,
     });
     log(chalk.green('Authenticated with Apple Developer Portal successfully!'));
 
     // Get all of the teams
     const teams = await Teams.getTeamsAsync();
```

Comment:
If auth context is extracted and should be passed to all functions that call apple API, why this function does not need it?

Reply:
this method creates the request context